qiita_db/sql_connection.py

```diff
@@ -255,33 +256,30 @@ def get_postgres_cursor(self):
         except PostgresError as e:
             raise QiitaDBConnectionError("Cannot get postgres cursor! %s" % e)
 
-    def set_autocommit(self, on_or_off):
-        """Sets the isolation level to autocommit or default (read committed)
```

Comment:
Would be good to move this docstring to the autocommit property. Otherwise this information is being lost.